src/test/java/org/broadinstitute/hellbender/tools/IndexFeatureFileIntegrationTest.java

```diff
@@ -490,4 +492,19 @@ public void testEnsemblGtfIndexQuery(final SimpleInterval interval,
             }
         }
     }
+
+    @Test
+    public void testNewGencodeLiftoverGtfFile() {
+        // First ensure that we can index the file:
+        // Required Args:
+        final ArgumentsBuilder arguments = new ArgumentsBuilder();
+
+        final File output = createTempFile(GENCODE_NEW_LIFTOVER_FILE.getName(), ".idx");
+
+        arguments.addInput(GENCODE_NEW_LIFTOVER_FILE);
+        arguments.addOutput(output.getAbsolutePath());
+
+        // Run the beast:
+        runCommandLine(arguments);
+    }
```

Comment:
What's the motivation behind this `IndexFeatureFile` test to index this specific gencode file? Can you add a code comment with an explanation?

Reply:
The latest gencode files for hg19 have slightly different header information and formatting
than previous versions (first noticed for gencode v34).  This is because they are lifting over the hg38 version to hg19.  This test is designed to ensure that such a file can still be indexed with the gencode codec.

I'll add this info to the test.